components/users.js

```diff
@@ -83,6 +83,27 @@ SteamCommunity.prototype.removeFriend = function(userID, callback) {
 	}, "steamcommunity");
 };
 
+SteamCommunity.prototype.getBlockedList = function (callback) {
+    this.httpRequestGet("https://steamcommunity.com/my/friends/blocked?ajax=1", function (err, res, body) {
+        if (res.req.path.substring(0,6) == "/login") {
+            callback(new Error("Not Logged In"));
+            return;
+        }
+        if (!body) {
+            callback(new Error("Malformed response"));
+            return;
+        }
+
+        var blocked = [],
+            regex=/\bdata-steamid="(\d+)"/g,
+            matches;
+        while (matches = regex.exec(body)) {
+            blocked.push( new SteamID(matches[1]) );
+        }
+        callback(null, blocked);
+    }, "steamcommunity")
+};
+
```

Comment:
This is probably not the correct place for this function, because all other functions are meant for a **single** user and take `userID` as argument.

Reply:
I think it fits best in `index.js`